core/src/io/anuke/mindustry/ui/fragments/BlockInventoryFragment.java

```diff
@@ -100,7 +101,7 @@ private void rebuild(boolean actions){
                         holding = false;
                         holdTime = 0f;
 
-                        Events.fire(new WithdrawEvent());
+                        Events.fire(new WithdrawEvent(tile, player, lastItem, amount));
```

Comment:
This will lead to duplicate withdraw events firing on the client when it is in singleplayer.